src/main/java/com/github/manolo8/darkbot/utils/LogUtils.java

```diff
@@ -90,8 +101,9 @@ private static void tryDelete(Path path) {
     }
 
     private static class PrintStreamWithDate extends PrintStream {
-        public PrintStreamWithDate(String logfile) throws FileNotFoundException, UnsupportedEncodingException {
-            super(logfile, "UTF-8");
+
+        public PrintStreamWithDate(OutputStream downstream) throws FileNotFoundException, UnsupportedEncodingException {
+            super(downstream, true, "UTF-8");
```

Comment:
You're assuming everyone's system works in UTF-8, avoid such assumption.

Java internally uses:
```
        setOut0(newPrintStream(fdOut, props.getProperty("sun.stdout.encoding")));
        setErr0(newPrintStream(fdErr, props.getProperty("sun.stderr.encoding")));
```

those `props` you can get via `System.getProperties().getProperty("sun.stdout.encoding")`.
If they're non-null, call with that encoding. If they're null, call a constructor without that encoding.

You can double-check all the java code in the System class:
![image](https://user-images.githubusercontent.com/11789291/208269197-b321b846-b7e3-4783-a018-6bf6185d0ab6.png)
![image](https://user-images.githubusercontent.com/11789291/208269200-2adc4594-50b9-4bfe-91e8-cca994bdb8b4.png)